test/tests/functional/pbs_resc_custom_perm.py

```diff
@@ -79,15 +60,13 @@ def setUp(self):
         # via qmgr -c 'create resource'
         attr = {'type': 'string', 'flag': 'h'}
         r = 'foo_str'
-        rc = self.server.manager(
-            MGR_CMD_CREATE, RSC, attr, id=r, runas=ROOT_USER, logerr=False)
-        self.assertEqual(rc, 0)
+        self.server.manager(
+            MGR_CMD_CREATE, RSC, attr, id=r, logerr=False)
 
         attr = {'type': 'string', 'flag': 'hm'}
         r = 'foo_strm'
-        rc = self.server.manager(
-            MGR_CMD_CREATE, RSC, attr, id=r, runas=ROOT_USER, logerr=False)
-        self.assertEqual(rc, 0)
+        self.server.manager(
+            MGR_CMD_CREATE, RSC, attr, id=r, logerr=False)
```

Comment:
Not sure, but you might be able to merge these back to 1 line since they are shorter now.

Reply:
It is done.